packages/management/core-management-plugins/bundle/config/telemetry.yaml

```diff
@@ -0,0 +1,29 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+
+---
+apiVersion: cli.tanzu.vmware.com/v1alpha1
```

Comment:
@anujc25 can you PTAL to see if this is impacted in any way by the package-based-lcm changes?